### PR TITLE
cargo-add: add page

### PR DESCRIPTION
**Pull request by marchersimon**
- **Version of the command being documented (if known):**  cargo 1.62.0

### CODE REVIEW

File: pages/common/cargo-add.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+`cargo add {{dependency}} --optional`
+
+- Add a local crate as a dependency:
+
+`cargo add --path {{path/to/crate}}`
+
+- Add a development or build dependency:
+
+`cargo add {{dependency}} --dev|--build`
```
Comment (navarroaxel) [inline pages/common/cargo-add.md:28]:
```suggestion
`cargo add {{dependency}} --{{dev|build}}`
```

Comment (marchersimon) [reply pages/common/cargo-add.md:28]: Hmm, most of our pages use the syntax you suggested, however I personally think it's not very clear. What do you think?

**Review by nicokosi — APPROVED** [review]
Looks good to me! 👍 🦀

File: pages/common/cargo-add.md (hunk 1 above)
Comment (navarroaxel) [reply pages/common/cargo-add.md:28]: IMHO `--dev|--build` is not clear enough.

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**